src/scripts/main.js

```diff
@@ -1,7 +1,23 @@
 'use strict';
 
 const pushNotification = (posTop, posRight, title, description, type) => {
-  // write code here
+  const messageWindow = document.createElement('div');
+
+  messageWindow.classList.add('notification', type);
+
+  messageWindow.style.top = `${posTop}px`;
+  messageWindow.style.right = `${posRight}px`;
+
+  messageWindow.innerHTML = `
+    <h2 class="title">${title}</h2>
+    <p>${description.replace('\n', '<br>')}</p>
+  `;
+
+  document.body.append(messageWindow);
+
+  setTimeout(() => {
+    document.body.remove(messageWindow);
+  }, 2000);
```

Comment:
Fix this
https://user-images.githubusercontent.com/61904995/231458140-b5a64b6e-9ac4-4988-8a6d-4d85cb3bd7d6.mp4
```suggestion
  setTimeout(() => {
    messageWindow .remove();
  }, 2000);
```